itests/qtest-iceberg/src/test/java/org/apache/hadoop/hive/cli/TestIcebergLlapLocalCompactorCliDriver.java

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
Because existing IcebergCli doesn't start a compaction worker.
When a compaction is run on a regular driver without compaction worker the tests hangs forever.
Hive ACID also has separate driver for compaction q-tests.